solutions/1630/01.py

```
@@ -0,0 +1,25 @@
+class Solution:
+    def checkArithmeticSubarrays(
+        self, nums: List[int], l: List[int], r: List[int]
```

Ruff (0.14.8)

Undefined name `List`

(F821)